snorkel/annotations.py

```diff
@@ -355,22 +355,31 @@ def load_matrix(matrix_class, annotation_key_class, annotation_class, session,
             kid_to_col[kid] = j
             col_to_kid[j]   = kid
 
-    # Create sparse matrix in LIL format for incremental construction
-    X = sparse.lil_matrix((len(cid_to_row), len(kid_to_col)), dtype=np.int64)
+    # Create sparse matrix in COO format for incremental construction
+    row = []
+    columns = []
+    data = []
 
     # NOTE: This is much faster as it allows us to skip the above join (which for some reason is
     # unreasonably slow) by relying on our symbol tables from above; however this will get slower with
     # The total number of annotations in DB which is weird behavior...
     q = session.query(annotation_class.candidate_id, annotation_class.key_id, annotation_class.value)
-    q = q.order_by(annotation_class.candidate_id)
```

Comment:
Can we make sure the matrix is ordered by candidate_id somewhere else though?  Maybe just tack this on to the query below?